llvm/test/CodeGen/AMDGPU/llvm.amdgcn.s.setprio.inc.wg.ll

```diff
@@ -0,0 +1,34 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
+; RUN: llc -march=amdgcn -mcpu=gfx1250 -show-mc-encoding -verify-machineinstrs < %s | FileCheck -check-prefix=GFX1250 %s
```

Comment:
I think it is recommended to use `-mtriple=amdgcn-amd-amdhsa` instead of `-march`. There was a bulk update in upstream in the past.

Reply:
Will do. This is a very old test of course.